promitor-agent-resource-discovery/values.yaml

```
@@ -71,11 +71,21 @@ resourceDiscoveryGroups: []
 #       - northeurope
 #       - westeurope
 
+deployment:
+  extraEnv: []
```

When looking at secrets, this might be more inline with that?
```suggestion
  env:
    extra: []
```

---

Done https://github.com/promitor/charts/pull/106/commits/4b93ed9cf8e4b3d662818672c0dc3f5d29decf9f